main.js

```diff
@@ -0,0 +1,71 @@
+/* eslint-disable max-classes-per-file */
+/* eslint-disable class-methods-use-this */
+/* eslint-disable new-cap */
+
+const form = document.getElementById('books');
+
+class books {
+  constructor(tittle, autor) {
+    this.tittle = tittle;
+    this.autor = autor;
+  }
+}
+
+class UI {
+  addbook(book) {
+    const productCard = document.getElementById('statusbooks');
+    const element = document.createElement('div');
+    element.innerHTML = `
+        <div>
+            <div class="bookstyle">
+                ${book.tittle}<br>
+                ${book.autor}<br>
+                <button type="button" name="deletebook">remove</button>
+                <hr class="styleline">
+            </div>
+        </div>
+        `;
+    productCard.appendChild(element);
+    this.resetform();
+  }
+
+  resetform() {
+    document.getElementById('books').reset();
+  }
+
+  removebook(element) {
+    if (element.name === 'deletebook') {
+      element.parentElement.parentElement.parentElement.remove();
+    }
+  }
+}
+
+function send(event) {
+  const tittlebook = document.getElementById('tittleBook').value;
+  const name = document.getElementById('nameAutor').value;
+  const book = new books(tittlebook, name);
+  const ui = new UI();
+  ui.addbook(book);
+
+  /* local storage */
+  if (localStorage.getItem('books') === null) {
+    const books = [];
+    books.push(book);
+    localStorage.setItem('books', JSON.stringify(books));
+  } else {
+    const books = JSON.parse(localStorage.getItem('books'));
+    books.push(book);
+    localStorage.setItem('books', JSON.stringify(books));
+  }
```

Comment:
- [ ] I noticed that when I reload the page all data disappear, after adding or deleting a book if the user reloads the page the data is supposed to be preserved in the browser's memory please kindly fix that 😉 🙃

Reply:
excuse me but i preserve the data in the web browser I send you a image  that is right 

![image](https://user-images.githubusercontent.com/107327344/191126301-e6f547d3-50b5-421e-8ad9-5aad2913c877.png)

Reply:
Hi @cvalencia1991, if it works that nice you can do other change and ask for the second review, your project will be approved 👍